packages/util-use-case/lib/adapters/toResult.test.mjs

```diff
@@ -0,0 +1,17 @@
+import test from '@zorko-io/tool-test-harness'
+import sinon from 'sinon'
+
+import {toResult} from './toResult'
+
+const result = {}
+const req = {}
+const res = {send: sinon.stub()}
+const deps = {}
+
+const toResultReturns = toResult(result, req, res, deps)
+
+test('Test toResult func', (t) => {
```

Comment:
`Test toResult func` it's too verbose, it makes sense to highlight that it's a simple scenario, for example`basic happy path`